### PR TITLE
Classifier

### PR DESCRIPTION
**Pull request by cskiz**
This re-creates the `classifier` branch, primarily using the `classifier-glob-entities`. `table.py` is added back along with other improvements.

### CODE REVIEW

**Review by cWren0110 — CHANGES_REQUESTED** [review]
Please update the merge conflict, and make sure file structure aligns with changes @rha930 made this morning.

File: src/text_classif/examples/predict_table.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
+"""
```
Comment (jgrundy) [inline src/text_classif/examples/predict_table.py:1]: Do we need to keep examples/predict_table.py? The version I've been using lives in utils/ now and I think this is a duplicate file.

Comment (cskiz) [reply src/text_classif/examples/predict_table.py]: You're right - I removed it.

**Review by jgrundy — APPROVED** [review]
Looks good. Mainly was paying attention to the import changes based on gc-ml splitting off, but it seems like you caught everything.

File: src/featurization/table.py [hunk 2]
```diff
@@ -7,14 +7,14 @@
 import numpy as np
 import pandas as pd
 
-import dataScience.src.utilities.spacy_model as spacy_
-from common.document_parser.issuance_ref import collect_ref_list
-from dataScience.src.featurization.abbreviations_utils import (
+import src.utilities.spacy_model as spacy_
```
Comment (cWren0110) [inline src/featurization/table.py:10]: Need to have replaced `dataScience` with `gamchanerml` .  For example: `import dataScience.src.utilities.spacy_model as spacy_` -> `import gamchanerml.src.utilities.spacy_model as spacy_`

File: README.md [hunk 3]
```diff
@@ -18,11 +18,6 @@ dataScience
 │   │   ├── evaluation
```
Comment (cWren0110) [inline README.md:18]: There is something weird going on with Github, but this file should be in conflict with what is currently in `dev` or at least showing changes were applied.  Specifically compare line 10 here and in the dev version.  There might be other changes I am not aware of as well.  Might be worth rebasing off of dev and re-dropping the contents of this PR in the appropriate location.

**Review by cWren0110 — CHANGES_REQUESTED** [review]
Anywhere where something was imported from `dataScience` should now import from `gamchanerml`. For example: import dataScience.src.utilities.spacy_model as spacy_ -> import gamchanerml.src.utilities.spacy_model as spacy_

File: README.md (hunk 3 above)
Comment (cskiz) [reply README.md:18]: Nothing weird on my end.  Anyway, `table.py` is up to date...
```
import src.utilities.spacy_model as spacy_
from src.featurization.ref_list import collect_ref_list
from src.featurization.abbreviations_utils import (
    get_agencies_dict,
    check_duplicates,
    get_agencies,
)
from src.featurization.extract_improvement.extract_utils import (
    extract_entities,
    create_list_from_dict,
    remove_articles,
    match_parenthesis,
)
```

Comment (cWren0110) [reply README.md:18]: On your line 10 of this file you have `dataScience` which no longer aligns with what is on `dev`, which is now `gamechangerml`

Additionally everything you pasted is improperly referenced according to my other change requests.

**Review by cWren0110 — CHANGES_REQUESTED** [review]

**Comment by cskiz** [comment]
That's Rob's README - should I update it for this PR to be approved?

**Comment by rha930** [comment]
all imports that are src should have gamechangerml infront i.e. gamechanger.src.
it looks like src is at the root level of the repo which is not the case

**Comment by cWren0110** [comment]
Yeah, that is what I am saying-- for some reason Github is not showing that the version you are submitting is in conflict/deviates from the one on `dev`, both of which are `gamechanger-ml/README.md` and so Github _should_ be catching the differences, but is not.  Thus my suggestion of "There might be other changes I am not aware of as well. Might be worth rebasing off of dev and re-dropping the contents of this PR in the appropriate location."

**Comment by cskiz** [comment]
> Yeah, that is what I am saying-- for some reason Github is not showing that the version you are submitting is in conflict/deviates from the one on `dev`, both of which are `gamechanger-ml/README.md` and so Github _should_ be catching the differences, but is not. Thus my suggestion of "There might be other changes I am not aware of as well. Might be worth rebasing off of dev and re-dropping the contents of this PR in the appropriate location."

Understand. My content source is `src`. I cloned, `pip install .` but `import gamechangerml` or variants fails. Replacing `dataScience` did not work. I'll see what I can do.

**Comment by cskiz** [comment]
I caught it. It seems I pulled `gamechanger-ml` prior to there being the `gamechanger` content root.

**Pull request closed without merging**